### ISSUE TITLE
Is Cypress Helper v2 actually related this?

### ISSUE THREAD

**Issue by rmaclean-ee** [90poe/vscode-cy-helper#63]
There is a [Cypress Helper v2](https://marketplace.visualstudio.com/items?itemName=shevtsov.vscode-cy-helper) which says it is a fork of this one?

Is that correct?

**Comment by Shelex** [comment]
Yes. This one was published with token based on work account and now I cannot do that anymore.
So in order to keep maintaining it I have forked this repo and published with personal token.
You can also check list of forks for this repo and check source github repo for Cypress Helper v2 in extensions marketplace.

**Comment by tnrich** [comment]
@Shelex does your new fork fix this issue: https://github.com/90poe/vscode-cy-helper/issues/61 

Thanks! :) Love your extension generally

**Comment by Shelex** [comment]
Hi @tnrich, thanks for feedback.
Oh, sorry, have missed that issue.
I guess that may be due to plugin initialisation. So when you start typing your snippet it is checking filesystem (reading package.json, locating test files, etc.) and may cause some delay in vscode response for snippets. Will check if it could be addressed.

Copied issue to fork.
